c7n/resources/route53.py

```diff
@@ -626,8 +631,17 @@ class resource_type(TypeInfo):
         name = id = 'ReadinessCheckName'
         global_resource = True
 
+    def __init__(self, ctx, data):
+        # Readiness check in Amazon Route53 ARC is global feature. However,
+        # US West (N. California) Region must be specified in Route53 ARC readiness check api call.
+        # Please reference this AWS document:
+        # https://docs.aws.amazon.com/r53recovery/latest/dg/introduction-regions.html
+        ctx.options['region'] = region
```

Comment:
we shouldn't modify the configuration options, we should instead pass region to `local_session(self.manager.session_factory, region=ARC_REGION)` or client as your doing below.

Reply:
yeah, I just modified local_client. That works better. Thanks.